aws/data_source_aws_redshift_service_account.go

```diff
@@ -45,6 +49,7 @@ func dataSourceAwsRedshiftServiceAccountRead(d *schema.ResourceData, meta interf
 
 	if accid, ok := redshiftServiceAccountPerRegionMap[region]; ok {
 		d.SetId(accid)
+		d.Set("arn", fmt.Sprintf("arn:%s:iam::%s:user/logs", meta.(*AWSClient).partition, accid))
```

Comment:
I don't think we should use fmt.Sprintf here - I think we should use the ARN package as part of AWS

Reply:
Will do